### DISCUSSION TITLE
How does the app work in general - comparing to other Apps like BTT

### DISCUSSION THREAD

**Discussion by pictosun** [rxhanson/Multitouch-Community#141] [General]
Hi,

hope this is the right place, to ask/discuss about the app.

I really like Multitouch and the development. Nearly everything within one small app. Great.

But I do have one question how this app works in general, because it's different from other apps. BTT or other Touch/Screen-Apps require access to security settings within macOS (different categories). But Multitouch doesn't ask/need access to them except accessibility settings.

What is the app doing different from other apps?

**Comment by rxhanson** [comment]
Thanks for the feedback and the question. 

I think the question can be rephrased as “what does the accessibility security setting in allow an app to do?”

In short, it allows an app to listen to input from devices and generate device input programmatically. Some other types of functionality fit under this permission as well, like managing windows, and all of Multitouch’s features are covered by it. 

There are some features that I have avoided, like screen capture features, in order to keep the app’s necessary permissions and feature set smaller.
